minwoo.lee_java/src/BOJ2015.java

```diff
@@ -0,0 +1,38 @@
+import java.io.*;
+import java.util.*;
+
+public class BOJ2015 {
+    public static void main(String[] args) throws IOException {
+        BufferedReader bf = new BufferedReader(new InputStreamReader(System.in));
+        StringTokenizer st = new StringTokenizer(bf.readLine());
+        HashMap<Integer, Integer> numSums = new HashMap<>();
+        int n, k;
+        long result;
+        int[] nums;
+        int[] prefix;
+        result = 0L;
+        n = Integer.parseInt(st.nextToken());
+        k = Integer.parseInt(st.nextToken());
+        nums = new int[n + 1];
+        prefix = new int[n + 1];
```

Comment:
일반적으로 자바에서는 필요한 곳에서 변수를 선언하고 초기화를 합니다. 
구글의 자바 코딩 스타일 가이드에서도 이를 이야기합니다.
(https://google.github.io/styleguide/javaguide.html#s4.8.2-variable-declarations 중 4.8.2.2 Declared when needed 파트)
그래서 선언과 초기화를 같이 하는게 좋아 보입니다.

```suggestion
        int n = Integer.parseInt(st.nextToken());
        int k = Integer.parseInt(st.nextToken());
        long result = 0L;
        int[] nums = new int[n + 1];;
        int[] prefix = new int[n + 1]; 
```